### ISSUE TITLE
Incorrect AuthorizationPolicy grants anybody access to endpoint.

### ISSUE THREAD

**Issue by bielawb** [ironmansoftware/universal-dashboard#1474]
#### Describe the Issue
If endpoint is configured to use AuthorizationPolicy that doesn't exist, it looks like no authorization is performed (so typo in the name will grant anybody access to the endpoint).

#### To Reproduce
1. Create UD Endpoint with non-existing authorization policy applied.
2. Profit.

#### Expected behavior
Possible outcomes:
- UD doesn't start
- UD starts, but endpoint fails with clear error message

#### Code sample
For some reason I couldn't repro using code below. We are running UD in IIS with -Wait, perhaps it surfaces itself only there. For sure I was able to use endpoints with incorrect authorizationPolicy there with unprivileged  account.

```powershell
$authPolicies = @(
    New-UDAuthorizationPolicy -Name MyPolicy -Endpoint {
        param ($ClaimsPrinciple)
        $ClaimsPrinciple.HasClaim(
            'http://schemas.microsoft.com/ws/2008/06/identity/claims/groupsid',
            $someSid
        )
    }
)

$correctEndpoint = New-UDEndpoint -AuthorizationPolicy MyPolicy -Endpoint {
    'Fails!'
} -Url /foo

$brokenEndpoint = New-UDEndpoint -AuthorizationPolicy SomeName -Endpoint {
    'Works!'
} -Url /bar

Start-UDDashboard -Name test -Port 31334 -Content {
    $Auth = New-UDAuthenticationMethod -Windows
    $LoginPage = New-UDLoginPage -AuthenticationMethod @($Auth) -AuthorizationPolicy $authPolicies -PassThru

    New-UDDashboard -Title WIA -Content { 
        New-UDRow -Columns {
            New-UDColumn -Size 12 -Endpoint {
                New-UDCard -Text "Logged in as $user"
            }
        }
    } -LoginPage $LoginPage
} -AllowHttpForLogin -Endpoint @(
    $correctEndpoint
    $brokenEndpoint
)
```

**Comment by McAndersDK** [comment]
So your running with nightly build? 
as of 2.8.2 new-udendpoint dont have the AuthorizationPolicy

**Comment by bielawb** [comment]
Can you point me to issue/ docs that explain how can I use authorizationPolicies without the parameter? I see it on all versions we use and looking at the open issues the plan was to add it to scheduled endpoints, not to remove it...? 

Also: missing parameter would most likely make dashboard crash, not produce unexpected behavior. My problem is not that it doesn't work at all, my problem is that it's allowing users to connect when we specify authorization policy that we didn't create.

**Comment by adamdriscoll** [comment]
@McAndersDK - It always had the parameter when using the REST API parameter set. It doesn't have it (until 2.8.3) for other parameter sets.

**Comment by adamdriscoll** [comment]
This has been resolved in our PowerShell Universal 1.2
